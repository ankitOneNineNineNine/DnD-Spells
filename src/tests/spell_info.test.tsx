import React from "react";
import { render } from "@testing-library/react";
import SpellInfo from "../pages/Spells/spell_info.page";
import { ProviderMock } from "../mocks/provider.mock";

describe("spell info", () => {
  it("should match a snapshot", () => {
    const { container } = render(
      <ProviderMock favorites={[]}>
        <SpellInfo />
      </ProviderMock>
    );
    expect(container).toMatchSnapshot();
  });
});
