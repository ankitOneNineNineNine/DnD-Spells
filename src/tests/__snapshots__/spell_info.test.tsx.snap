// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spell info should match a snapshot 1`] = `
<div>
  <div>
    <div
      class="cardContainer"
    >
      <h1 />
      <h3 />
      <p>
        <span>
          Casting Time: 
        </span>
      </p>
      <p>
        <span>
          Components: 
        </span>
      </p>
      <p>
        <span>
          Attack Type: 
        </span>
        Not Mentioned
      </p>
      <p>
        <span>
          Classes That Can Learn Spell : 
        </span>
      </p>
      <p>
        <span>
          Level: 
        </span>
         
      </p>
      <p>
        <span>
           Range of Spell: 
        </span>
      </p>
      <p>
        <span>
          Magic School: 
        </span>
      </p>
      <p>
        <span>
          Higher Level Spell: 
        </span>
        None
      </p>
      <h3>
        Description
      </h3>
      <ul
        style="list-style-position: inside;"
      />
    </div>
  </div>
</div>
`;
